### PR TITLE
Tips IA: Create better title/subtitle

### PR DESCRIPTION
**Pull request by gaulrobe**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] Improvement
  - [x] Enhancement: **`Tips: Create a better title/subtitle`**
###### Description of changes

The previous title, which was pretty verbose, is now the subtitle as it does a better job explaining context of the tip calculation. The new title is now the succinct answer to the query. Per a discussion in #3208.

I changed the percentage answering to this format as well even though it'll be eventually removed. This was to make the tests easier to write and the removal of said code to be (hopefully) easier when the time comes.
###### People to notify (@mention interested parties)

@mattlehning @moollaza

---

Instant Answer Page: https://duck.co/ia/view/tips

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mattlehning

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gaulrobe** [comment]
@moollaza ping!

**Comment by moollaza** [comment]
@gaulrobe this looks awesome! 💯 

One thought is to re-iterate the percentage in the subtitle:

>**$6.88**
>Subtotal: $45.84 | 15% Tip: $6.88 | Total: $52.72

File: lib/DDG/Goodie/Tips.pm [hunk 1]
```diff
@@ -46,10 +47,11 @@ handle query_lc => sub {
     my $number = $style->for_display($num);
     my $percent_answer = "$sign$calculated_answer is $percentage% of $sign$number";
     
-    return $percent_answer,
+    return "$sign$calculated_answer",
         structured_answer => {
             data => {
-                title => "$percent_answer"
+                title => "$sign$calculated_answer",
+                subtitle => "$percent_answer"
```
Comment (moollaza) [inline lib/DDG/Goodie/Tips.pm]: If you return an array of strings here, we'll automatically separate them. The output should look like:
15__tip_on_45_84_at_duckduckgo

I think you can `split /;/, $percent_answer;`

Comment (gaulrobe) [inline lib/DDG/Goodie/Tips.pm]: Awesome to know! I'll do that :+1:

**Comment by mintsoft** [comment]
Hiya @gaulrobe Just checking in, are you OK with the tweaks that are suggested here and still interested in finishing it off?

**Comment by moollaza** [comment]
This PR is being closed due to inactivity. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**